config/outline.go

```diff
@@ -233,7 +233,7 @@ func (outline Outline) GetServers() (servers []Server, err error) {
 
 type AccessKey struct {
 	ID               string `json:"id"`
-	Name             string `json:"name"`
+	Name             string `json:"name"` // the server name in outline
```

Comment:
This comment is incorrect. The `Name` here is access key username.

Reply:
What is the username? Could you please give me an example?

Reply:
Feel free to correct it.